test/automation/navigation.tests.ps1

```diff
@@ -104,7 +104,7 @@ AfterEach{
        $h[0].Name | Should Be "Bill"
     }
 
-    It "cd to root for non-Windows" -Skip:($IsWindows){
+    It "cd to root for non-Windows" -Skip:($script:IsWindows){
```

Comment:
Why are there Windows and non-Windows versions of this test? You should be able to use the Path fields and methods to write it once and have it run everywhere in a cross-platform manner.

Reply:
In test, I do not wish to concatenate using path for some cases like `cd ..\..\`,   'cd ../../`,  `cd az:/`, `cd az:\`.   I would rather directly check for the correctness.

Reply:
But the code is almost exactly the same modulo "/" and "\". Writing a second test is just duplicate code and unnecessary

Reply:
My comment is specific to the test